src/ui/components/SidePanel.tsx

```diff
@@ -92,25 +102,44 @@ export default function SidePanel() {
   const { recording } = useGetRecording(recordingId);
   const testSuite = recording && isTestSuiteReplay(recording);
 
+  const { comments, loading } = hooks.useGetComments(recordingId);
+
+  const [isNGXHeaderVisible, setIsNGXHeaderVisible] = useState(
+    localStorage.getItem("NGXHeaderVisibility") !== "hidden"
+  );
+
+  const initialPanelRef = useRef(selectedPrimaryPanel);
+
+  useEffect(() => {
+    if (selectedPrimaryPanel !== initialPanelRef.current) {
+      setIsNGXHeaderVisible(false);
+      localStorage.setItem("NGXHeaderVisibility", "hidden");
```

Comment:
What is "NGX" ? I thought this was "NUX"

Reply:
I think it's "New Guest Experience", where "Guest" === "anonymous user who hasn't signed up yet".